modules/cnf-reducing-netqueues-using-pao.adoc

```diff
@@ -0,0 +1,346 @@
+
+* `userLevelNetworking` is a required field that specifies a boolean flag which, if `true` sets the queue count to the reserved CPU count for all supported devices. The default is `false`.
+* `Devices` is an optional field which specifies a list of devices that will have the queues set to the reserved CPU count. If the device list is empty the configuration applies to all network devices. The configuration is as follows:
+** `interfaceName`: Name supports shell-style wildcards which can be positive or negative.
+*** Example wildcard syntax is as follows: `<string> .*`
+*** Negative rules are prefixed with an exclamation mark. To apply the net queue changes to all devices other than the excluded list, use  `!<device>`; for example, `!eno1`.
+* `vendorID`: Network device vendor ID represented as a 16 bit hexmadecimal number with a 0x prefix.
+* `deviceID`: Network device ID (model) represented as a 16 bit hexmadecimal number with a 0x prefix.
++
+[NOTE]
+====
+When a `deviceID` is specified, the `vendorID` must also be defined. A device that matches all of the device identifiers specified in a device entry `interfaceName`, `vendorID` or a pair of `vendorID` plus `deviceID` representing a network device qualifies as a network device that will have its net queues count set to the reserved CPU count.
+
+When two or more devices are specified, the net queues count is set to any net device that matches one of them.
+====
++
+An example performance profile configuration is shown below:
++
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+  kind: PerformanceProfile
+  metadata:
+    name: manual
+  spec:
+    cpu:
+    isolated: 3-51,54-103
+    reserved: 0-2,52-54
+
+    net:
+    - userLevelNetworking: true         <1>
+    #!
+
+    more examples:
+
+      net:
+        userLevelNetworking: true
+        devices:
+            -interfaceName: “eth0”
+            -interfaceName: “eth1”      <2>
+            -vendorID: “0x1af4”
+             deviceID: “0x1000”
+
+      net:
+       userLevelNetworking: true        <3>
+       devices:
+        -interfaceName: “eth*”
+
+      net:
+       userLevelNetworking: true        <4>
+       devices:
+        -interfaceName: “!eno1”
+
+     net:
+       userLevelNetworking: true        <5>
+       devices:
+        -interfaceName: “eth0”
+         vendorID: “0x1af4”
+         deviceID: “0x1000”
+
+    #!
+
+      nodeSelector:
+          node-role.kubernetes.io/worker-cnf: ""
+
+----
+Set the queue count to the reserved CPU count for:
+<1>  All devices.
+<2>  All devices that match any of the defined device identifiers.
+<3>  All devices starting with the interface name `eth`.
+<4>  All devices with an interface named anything other than `eno1`.
+<5>  All devices that have an interface name `eth0`, `vendorID` 0x1af4, and `deviceID` 0x1000.
+
+. Apply the performance profile.
++
+[source,terminal]
+----
+$  oc apply -f <your_profile_name>.yaml
```

Comment:
Doesn't step 2 have you doing an `oc edit`? Wouldn't you just save that file then, and not apply a file (I don't see a file being created in the procedure)?